### ISSUE TITLE
5.3. Fine-grained Recognition

### ISSUE THREAD

**Issue by xiaohe725** [zeiss-microscopy/BSConv#12]
batch_size:128,   momentum:0.9 ,weight decay  10−4.epochs:100,learning rate：0.1, linearly decayed at every epochs.
The results of the paper cannot be reproduced. Is there a problem with my hyperparameter setting
